### DISCUSSION TITLE
I want flexbox sizing to factor in the width of wrapped text

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#26] [Wants]
I've made this code pen to demonstrate and explain the issue more deeply:

https://codepen.io/daniel-tonon/pen/VwLmqvb

The best work around that I can think of for this is to apply a fixed width container around the content of each problematic flex item. The fixed width container would be different for each item and each items width would need to be carefully managed with lots of media queries. I managed to avoid this problem in the demo by not making the demo responsive. 

Applying fixed widths to flexbox items goes against the spirit of what flexbox is trying to do. Also, the instant any of the content changes, all of the media query and fixed width CSS settings would need updating.

See: https://webwewant.fyi/wants/5e52722b13def22bf8bfd348/

**Comment by bkardell** [comment]
see also https://github.com/w3c/csswg-drafts/issues/4801 /
https://github.com/w3c/csswg-drafts/issues/191 ? should they be linked
maybe?

On Thu, Feb 27, 2020 at 3:31 PM Aaron Gustafson <notifications@github.com>
wrote:

> https://webwewant.fyi/wants/5e52722b13def22bf8bfd348/
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WebWeWant/webwewant.fyi/issues/26?email_source=notifications&email_token=AAGUQZP5MJ4XAPZFOP6V5X3RFAPKNA5CNFSM4K5A4LJKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENF26LA#issuecomment-592162604>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGUQZKSZ5B5YEARCVVBNMDRFAPKNANCNFSM4K5A4LJA>
> .
>


-- 
Brian Kardell :: @briankardell :: bkardell.com

**Comment by Dan503** [comment]
Thanks for looping my in.

After sending the email, I later discovered that this was essentially a duplicate of this already existing, and somewhat larger CSS Working group issue:

https://github.com/w3c/csswg-drafts/issues/191
